en/vespa8-release-notes.html

```diff
@@ -574,6 +574,18 @@ <h3 id="strict-mode">Strict mode enabled by default</h3>
   attribute in services.xml for details.
 </p>
 
+<h3 id="request-headers-for-remote-in-access-log">Request headers controlling remote host/port in access log</h3>
+<p>
+  The jdisc container will use the <em>X-Forwarded-For</em> and <em>X-Forwarded-Port</em> request headers to set the remote
+  host and port respectively in the access log. The following request headers will no longer be handled by default:
+</p>
+<ul>
+  <li>y-ra</li>
+  <li>yahooremoteip</li>
```

Comment:
Do the 'y' headers belong in the public doc?

Reply:
They shouldn't have been part of open-source Vespa, but they did.